graph_adventure/adv.py

```diff
@@ -13,7 +13,30 @@
 
 
 # FILL THIS IN
-traversalPath = ['s', 'n']
+traversalPath = []
+
+def dft_visit_all_rooms(self, starting_vertex):
+    visited = {}
+    backtrack = []
+    move_counter = []
+    while len(list(visited)) < 499:
+        #while there is a current path forward
+        #add unvisited to visited
+        if player.currentRoom.id not in visited:
+
+    #if you reach a dead end, back track until there is an unvisited path forward
+    while
+\
+# {0: {'n': '?', 's': '?', 'w': '?', 'e': '?'}}
+    
+    #rooms are stacked, backtracking is recursion
```

Comment:
your pseudo code is really on point, but I would like for you to attempt to redefine your pseudo code into actual syntax